MapboxVision/src/main/java/com/mapbox/vision/sync/syncmanager/TelemetrySyncManager.kt

```diff
@@ -0,0 +1,279 @@
+package com.mapbox.vision.sync.syncmanager
```

Comment:
Was it changed from `Impl` or just factored out here?

Reply:
try to leave as is.
please check